shell/package.json

```diff
@@ -12,9 +12,9 @@
     "*.less"
   ],
   "scripts": {
-    "test": "jest",
-    "test:coverage": "jest --verbose --coverage",
-    "test:watch": "jest --verbose --watch",
+    "test": "../node_modules/.bin/jest",
```

Comment:
maybe this is not required when run `pnpm test`

Reply:
<img width="470" alt="image" src="https://user-images.githubusercontent.com/3955437/173540704-5d82a594-7e50-4596-af7a-a08c3f19e93b.png">